packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Checkmarks.Example.tsx

```diff
@@ -25,198 +24,188 @@ const keys: string[] = [
   'disabledPrimarySplit'
 ];
 
-export class ContextualMenuCheckmarksExample extends React.Component<{}, IContextualMenuMultiselectExampleState> {
-  constructor(props: {}) {
-    super(props);
+export const ContextualMenuCheckmarksExample: React.FunctionComponent = () => {
+  const [selection, setSelection] = React.useState<{ [key: string]: boolean }>({});
 
-    this._onToggleSelect = this._onToggleSelect.bind(this);
+  const onToggleSelect = (ev?: React.MouseEvent<HTMLButtonElement>, item?: IContextualMenuItem): void => {
+    ev && ev.preventDefault();
 
-    this.state = {
-      selection: {}
-    };
-  }
+    if (item) {
+      console.log(selection);
+      setSelection({ ...selection, ...{ [item.key]: selection[item.key] === undefined ? true : !selection[item.key] } });
```

Comment:
The second spread isn't really needed here.
```suggestion
      setSelection({ ...selection, [item.key]: selection[item.key] === undefined ? true : !selection[item.key] });
```